### DISCUSSION TITLE
Why is a `winit` fork used?

### DISCUSSION THREAD

**Discussion by aentity** [hecrj/iced#960] [Q&A] [answered]
this crate not usable outside of itself right now because it uses fork of winit. why is fork used? even better, when can we use winit 0.25 on crates please? is there PR i can make to do this?

**Comment by hecrj** [comment, marked as answer]
We forked `winit` to extend its functionality and satisfy our use cases without introducing an additional burden to its maintainers. It's a highly experimental fork, and we plan to contribute the changes back to `winit` once we have confidence in them.

An example of a feature we landed thanks to this is system menus (see #945).
